src/documents/ComponentGrades.md

```diff
@@ -14,121 +14,87 @@ The Infusion Framework already contains several predefined component grades that
     <thead>
         <tr>
             <th>Grade Name</th>
-            <th>"Relay" Version <sup>[1]</sup></th>
             <th>Description</th>
         </tr>
     </thead>
     <tbody>
         <tr>
-            <td>autoinit</td>
-            <td></td>
+            <td><code>fluid.component</code></td>
             <td>
-                A special directive grade that instructs the framework to automatically construct a globally named creator function (with the same name as the grade) responsible for the construction of the component. NOTE: for the Infusion 2.0 release this grade will become redundant as it will be the default for every grade
-                <br/>
-                <em>
-                    <strong>NOTE:</strong> for the Infusion 2.0 release this grade will become redundant as it will be the default for every grade
-                </em>
+                A "little" component is the most basic component: it supports options merging with defaults (<a href="tutorial-gettingStartedWithInfusion/BasicComponentCreation-LittleComponents.md">Little Components</a>), as well as
```

Comment:
And should we even still be calling these "little" components now that they've been renamed? I'd recommend we ditch the prefix and rename the appropriate documentation pages.